playbooks/roles/edxapp/defaults/main.yml

```diff
@@ -182,16 +182,69 @@ EDXAPP_PDF_RECEIPT_TAX_ID_LABEL: "fake Tax ID"
 EDXAPP_PDF_RECEIPT_COBRAND_LOGO_PATH: ""
 EDXAPP_PDF_RECEIPT_LOGO_PATH: ""
 
+COURSE_ABOUT_VISIBILITY_PERMISSION: 'see_about_page'
+COURSE_CATALOG_VISIBILITY_PERMISSION: 'see_in_catalog'
```

Comment:
Wait, this doesn't look correct w/out an `EDXAPP_` prefix.
Can you look into this?